server.js

```diff
@@ -221,13 +231,12 @@ var renderChart = (request, window, callback) => {
     .attr('class', 'y grid')
     .call(yGrid);
 
-  var xAxis = d3.svg.axis()
+  var xAxis = xTickStrategy(d3.svg.axis()
```

Comment:
don't like this being wrapped around the call. would prefer if it was called something like `applyTickStrategy` and we call it after as a separate function call